CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This document follows the conventions laid out in [Keep a CHANGELOG](https://kee
 
 ### Added
 - Add CODEOWNERS file. [#24](https://github.com/lumicks/doltcli/pull/24)
+- Add remote and all options to branch command. [#20](https://github.com/lumicks/doltcli/pull/20)
```

Comment:
```suggestion
- Add `remote` and `all` options to branch command. [#20](https://github.com/lumicks/doltcli/pull/20)
```
it looks like broken english otherwise :D